tests/robot-tests/tests/admin_and_public/bau/publish_methodology.robot

```diff
@@ -381,6 +427,17 @@ Schedule a methodology amendment to be published with a release amendment
     ...    publishing_strategy=WithRelease
     ...    with_release=${PUBLICATION_NAME} - ${RELEASE_NAME}
 
+Check methodology status history table contains new row after approving amendment
```

Comment:
Do we check in any tests that we're not accidentally creating more status history items if we're *not* approving it, out of interest?  E.g. Draft => Draft, or Approved => Draft?

Reply:
I would expect it to create a new status history entry if it was moved from Approved => Draft, but not if it was moved from Draft => Draft or Approved => Approved, and with my manual testing (I believe) that is how it currently works. This is why I removed the InternalReleaseNote check from `IsStatusUpdateRequired`.

I don't think there are any tests around this at the moment. But this is deliberate as I didn't want to write a load of tests but then have to rewrite them once the Higher Review (EES-4376) work is done. I believe the risk in the mean time is minimal - I'm confident it works, but even if it didn't it would be a super easily fix, if we even considered a priority to fix.

All that said, happy to write more tests if you'd prefer.